src/main/java/natsu/command/UnmarkCommand.java

```diff
@@ -0,0 +1,18 @@
+package natsu.command;
+
+import natsu.exception.InvalidCommandException;
+
+import static natsu.util.Printer.printTaskUndone;
+import static natsu.util.TaskManager.list;
+
+public class UnmarkCommand {
+    public UnmarkCommand(String userInput) throws InvalidCommandException{
```

Comment:
There needs to be a gap between InvalidCommandException and { as per coding standards